src/components/breakdownItem.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function BreakdownItem() {
```

Comment:
since it took us a min to figure out, may want to note this is for the "found on" list of guns